contact.html

```diff
@@ -22,23 +24,24 @@
 		</div>
 		<div class="contactdetails">
 			<div class="contacticons">
-			<div class="mail"><a href="mailto:nickclifford@gmail.com">
-				<img alt="Email" src="mail.png" width="75" height="75">
+			<div id="mail"><a href="mailto:nickclifford@gmail.com">
```

Comment:
If you add e.g.
```suggestion
			<a href="mailto:nickclifford@gmail.com" target="_blank">
```
Your link will open up in a new tab